src/main/scala/zio/prelude/Associative.scala

```diff
@@ -89,14 +89,13 @@ object Associative extends Lawful[AssociativeEqual] {
     }
 
   /**
-   * The `Commutative` and `Inverse` instance for the product of `Boolean`
+   * The `Commutative` and `Identity` instance for the product of `Boolean`
    * values.
    */
-  implicit val BooleanProdCommutativeInverse: Commutative[Prod[Boolean]] with Identity[Prod[Boolean]] =
```

Comment:
this would then need to be `Inverse[Prod[Boolean]]`

Reply:
I just thought for consistency we should remove the `Inverse` instances for `Prod`. This behavior is already described by boolean conjunction and since we don't have any other instances of `Inverse[Prod[_]]` anymore it doesn't really make sense to try to abstract over that type.

Reply:
I think if it doesn't violate any laws, then it should be perfectly fine to keep it. I'm sure some users will find it useful for something.
(And we're keeping `Inverse[Prod[_]]` for floats anyway, right?)

Reply:
No, I got rid of those. I don't think those are lawful because` 0.0 / 0.0 != 0.0`.